api/relationships.go

```diff
@@ -68,7 +68,7 @@ func (entity *RelationshipsMarital) Valid() (bool, error) {
 
 	sv := entity.Status.Value
 	switch {
-	case sv == "InCivilUnion" || sv == "Separated":
+	case sv == "Separated":
```

Comment:
It seems like there is a bug in the existing code – what do you think? Should be doing something for "Married" value at first glance.

Reply:
Yeah on second look I think you're right, I'll add `Married` in since I don't see it being evaluated anywhere else in that switch statement.